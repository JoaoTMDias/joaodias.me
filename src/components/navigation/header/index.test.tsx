import React from "react";
import { render, cleanup } from "@testing-library/react";

import { Header } from "./index";

afterEach(cleanup);

describe("<Header />", () => {
	it("should render without errors", () => {
		const component = render(<Header />);

		expect(component).toMatchSnapshot();
	});
});
