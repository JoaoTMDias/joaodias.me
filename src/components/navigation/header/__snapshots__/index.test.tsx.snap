// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render without errors 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="sc-eqIVtm cQqwyE header"
        id="page-header"
      >
        <div
          class="layout__row header__row"
        >
          <a
            aria-current="page"
            aria-label="Press Enter/Click to go to the homepage."
            class="sc-cSHVUG ezoGnA"
            href="/"
            id="branding"
          >
            <svg
              class="sc-kAzzGY gQbpWy"
              height="48"
              id="logo"
              viewBox="0 0 48 48"
              width="48"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                class="brand__circle"
                cx="24"
                cy="24"
                fill="transparent"
                r="24"
              />
              <path
                class="brand__hair"
                d="M26.09 23.19s1.52-.6-.56 3.06c0 0 3.79-.87 3.11-3.94.22 1 .52.4-.12 3a4.73 4.73 0 0 0 2.65-2.65c0 .32.8.81-.33 2.9.56-.2.72-.39.85-.85a8.21 8.21 0 0 1-.43 3.74 5.18 5.18 0 0 0 1.67-2.3c.13-.92.77 3.37-.57 6.18a11.49 11.49 0 0 0 1.41-1.78c.56-1.22 0 2.84-.1 4.28-.49 2 .25 2 .25 2.92a3.73 3.73 0 0 0 1.59.12 2.87 2.87 0 0 0 1.83-1.34 6.75 6.75 0 0 0 .85-1.59.93.93 0 0 1-.61-.12c-.37-.24-.37-1-.37-1 2.07 1.15 2.37-.32 2.06-1.59a2 2 0 0 1-.27-1.44c.64-16.33-6.7-16.46-6.7-16.46a9 9 0 0 1 2.92-1.12 9.81 9.81 0 0 0-3.91 1.25 3.46 3.46 0 0 0-3-3.17A2.5 2.5 0 0 1 29.88 13s-5.53-4.33-11.46-1.84a9.4 9.4 0 0 1 4.77 0c-.19-.1-10.42-.95-14.55 8.13 1.6-1.77 2.15-2.4 2.46-2.16.19 0-2.23 3.58-2 12.93.15 1.47.5 1.25.73 1.71-.12.34-.35.4-.49.85-1.59 3 2.69 3.65 2.69 3.65a1.86 1.86 0 0 1-.37-2.44H12a2.15 2.15 0 0 1 .85.61c0-.37-.6-3.7 0-4.1 12.24-3.59 13.24-7.15 13.24-7.15zm-13.23 13.5a.6.6 0 0 1 .61-.12.62.62 0 0 1 .39.49v.12a3.81 3.81 0 0 1-2 .73c-.73 0-.59-.49-1.07-1.1.02 0 1.58.56 2.07-.12z"
              />
              <path
                class="brand__glasses"
                d="M12.11 27.91a.87.87 0 0 0 .26.7l.1.12a1.12 1.12 0 0 1 .12.35c.5 3 .61 3.42 1.87 4.08a7.57 7.57 0 0 0 3.09.53 7.89 7.89 0 0 0 3.52-.69c1.69-.88 2.33-3.94 2.36-4.09s.05-.38.83-.38c.73 0 .79.36.79.4s.66 3.18 2.35 4.07a8 8 0 0 0 3.6.7 7.56 7.56 0 0 0 3.08-.53c1.26-.66 1.37-1.07 1.87-4.08a1.12 1.12 0 0 1 .12-.35l.1-.12a.87.87 0 0 0 .26-.7V27a1 1 0 0 0-.85-.9 22.37 22.37 0 0 0-6-.3h-.11a8.09 8.09 0 0 0-3 .54l-.6.29a5.77 5.77 0 0 1-1.07.41l-.3.06a1.57 1.57 0 0 1-.53 0l-.3-.06a5.77 5.77 0 0 1-1.07-.41l-.6-.29a8.09 8.09 0 0 0-3-.54 22.37 22.37 0 0 0-6 .3 1 1 0 0 0-.85.9v.88zm23.79-.75v.45l-.56-.22zm-9.72.65c.08-.21.62-.91 3.93-1.1.58 0 1.07-.05 1.51-.05 2 0 2.58.35 2.95.92a3.26 3.26 0 0 1 .37 1.5c0 1-.19 2.73-1.55 3.38a6.12 6.12 0 0 1-2.5.48c-1.81 0-3.25-.61-3.87-1.64-1.29-2.16-.85-3.45-.84-3.48zm-12.24-.24c.37-.57 1-.92 3-.92.43 0 .92 0 1.51.05 3.31.19 3.85.89 3.94 1.13 0 0 .44 1.3-.85 3.46a4.44 4.44 0 0 1-3.87 1.64 6.12 6.12 0 0 1-2.5-.48c-1.36-.66-1.56-2.4-1.55-3.38a3.39 3.39 0 0 1 .32-1.5zm-1.34-.4l.56.23-.56.22z"
              />
            </svg>
          </a>
          <nav
            aria-label="Back Navigation Button"
            class="sc-bwzfXH iZKGMu"
            id="back-navigation"
          >
            <a
              aria-current="page"
              aria-label="Go back to the home page"
              class=""
              href="/"
            >
              <svg
                aria-labelledby="back-navigation-title"
                viewBox="0 0 48 48"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title
                  id="back-navigation-title"
                >
                  Navigate Back to the Homepage
                </title>
                <path
                  class="back__navigation__wrapper"
                  d="M0 0h48v48H0z"
                  fill="none"
                />
                <path
                  class="back__navigation__arrow"
                  d="M40 22H15.66l11.17-11.17L24 8 8 24l16 16 2.83-2.83L15.66 26H40z"
                  fill="var(--back-navigation-fill)"
                />
              </svg>
            </a>
          </nav>
          <h2
            class="sc-bdVaJa yzgbS"
            id="header-page-title"
          >
            Project Details
          </h2>
          <nav
            class="sc-fBuWsC enoLqX"
            id="top-navigation"
            title="Top Navigation"
          >
            <ul
              class="sc-fMiknA cHkchB"
            >
              <li
                class="sc-dVhcbM hWIHTP"
              >
                <a
                  aria-label="Go to the about page"
                  class="large-nav__link"
                  href="/about/"
                >
                  About
                </a>
              </li>
              <li
                class="sc-dVhcbM hWIHTP"
              >
                <a
                  aria-label="Go to the contacts page"
                  class="large-nav__link"
                  href="/contact/"
                >
                  Contact
                </a>
              </li>
            </ul>
          </nav>
          <nav
            class="sc-jlyJG iLFoRH"
            id="social-navigation"
            title="Social network links"
          >
            <ul
              class="sc-gipzik dClvSF"
            >
              <li
                class="sc-csuQGl exHRHf"
              >
                <a
                  aria-label="Follow me on Facebook"
                  class="sc-Rmtcm ghqkuW"
                  href="https://fb.com/JoaoTMDias"
                  rel="noopener noreferrer"
                  tabindex="0"
                  target="_blank"
                  to="https://fb.com/JoaoTMDias"
                >
                  <svg
                    viewBox="0 0 24 24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M12.88 22v-.07a10.55 10.55 0 0 1-1.13.07z"
                      fill="white"
                    />
                    <path
                      class="social__fill"
                      d="M15.12 13l1.13-2.81h-3.37V7.94A1.13 1.13 0 0 1 14 6.81h1.12V4h-2.24A3.38 3.38 0 0 0 9.5 7.38v2.81H7.25V13H9.5v8.74a10.41 10.41 0 0 0 2.25.26 10.55 10.55 0 0 0 1.13-.07V13z"
                      fill="var(--color-gray5, #afb2be)"
                    />
                  </svg>
                </a>
              </li>
              <li
                class="sc-csuQGl exHRHf"
              >
                <a
                  aria-label="Follow me on Instagram"
                  class="sc-Rmtcm ghqkuW"
                  href="https://instagram.com/JoaoTMDias"
                  rel="noopener noreferrer"
                  tabindex="0"
                  target="_blank"
                  to="https://instagram.com/JoaoTMDias"
                >
                  <svg
                    viewBox="0 0 24 24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      class="social__fill"
                      d="M15.75 2h-7.5A6.25 6.25 0 0 0 2 8.25v7.5A6.25 6.25 0 0 0 8.25 22h7.5A6.25 6.25 0 0 0 22 15.75v-7.5A6.25 6.25 0 0 0 15.75 2zm4.37 13.75a4.37 4.37 0 0 1-4.37 4.37h-7.5a4.37 4.37 0 0 1-4.37-4.37v-7.5a4.37 4.37 0 0 1 4.37-4.37h7.5a4.37 4.37 0 0 1 4.37 4.37z"
                      fill="var(--color-gray5, #afb2be)"
                    />
                    <path
                      class="social__fill"
                      d="M12 7a5 5 0 1 0 5 5 5 5 0 0 0-5-5zm0 8.12A3.12 3.12 0 1 1 15.12 12 3.12 3.12 0 0 1 12 15.12z"
                      fill="var(--color-gray5, #afb2be)"
                    />
                    <circle
                      cx="17.38"
                      cy="6.62"
                      fill="var(--color-gray5, #afb2be)"
                      r=".67"
                    />
                  </svg>
                </a>
              </li>
              <li
                class="sc-csuQGl exHRHf"
              >
                <a
                  aria-label="Check out my repos on Github"
                  class="sc-Rmtcm ghqkuW"
                  href="https://github.com/Joaotmdias"
                  rel="noopener noreferrer"
                  tabindex="0"
                  target="_blank"
                  to="https://github.com/Joaotmdias"
                >
                  <svg
                    viewBox="0 0 24 24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      class="social__fill"
                      d="M12 2.21a10 10 0 0 0-3.16 19.56c.5.1.68-.21.68-.48v-1.71c-2.78.61-3.37-1.34-3.37-1.34A2.64 2.64 0 0 0 5 16.77c-.91-.62.07-.61.07-.61a2.1 2.1 0 0 1 1.53 1 2.13 2.13 0 0 0 2.94.84 2.22 2.22 0 0 1 .63-1.35c-2.22-.25-4.55-1.11-4.55-5A3.89 3.89 0 0 1 6.65 9a3.62 3.62 0 0 1 .1-2.66s.84-.27 2.75 1a9.92 9.92 0 0 1 2.5-.28 10 10 0 0 1 2.51.34c1.9-1.3 2.74-1 2.74-1a3.56 3.56 0 0 1 .1 2.6 3.89 3.89 0 0 1 1 2.69c0 3.86-2.34 4.71-4.57 5a2.39 2.39 0 0 1 .68 1.86v2.75c0 .33.18.58.69.48A10 10 0 0 0 12 2.21z"
                      fill="var(--color-gray5, #afb2be)"
                    />
                  </svg>
                </a>
              </li>
              <li
                class="sc-csuQGl exHRHf"
              >
                <a
                  aria-label="Check out some dribble shots"
                  class="sc-Rmtcm ghqkuW"
                  href="https://dribbble.com/Joaotmdias"
                  rel="noopener noreferrer"
                  tabindex="0"
                  target="_blank"
                  to="https://dribbble.com/Joaotmdias"
                >
                  <svg
                    viewBox="0 0 24 24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      class="social__fill"
                      d="M12 2a10 10 0 1 0 10 10A10 10 0 0 0 12 2zm0 18.75A8.75 8.75 0 1 1 20.75 12 8.77 8.77 0 0 1 12 20.75z"
                      fill="var(--color-gray5, #afb2be)"
                    />
                    <path
                      class="social__fill"
                      d="M16.79 6.23A7.47 7.47 0 0 0 12 4.5a7 7 0 0 0-1.53.16 36 36 0 0 1 2.46 4 18 18 0 0 0 3.86-2.43zM14.39 11.88a13.87 13.87 0 0 1 5.1-.09 7.44 7.44 0 0 0-1.81-4.68 19 19 0 0 1-4.21 2.68c.33.69.64 1.38.92 2.09zM9.21 5a7.48 7.48 0 0 0-4.51 5.31 17.6 17.6 0 0 0 7.06-1.16A34.25 34.25 0 0 0 9.21 5zM12.33 10.29a18.92 18.92 0 0 1-6.83 1.3h-1v.45a7.46 7.46 0 0 0 1.77 4.83 14.05 14.05 0 0 1 6.9-4.64c-.26-.68-.54-1.32-.84-1.94zM14.85 13.06a35.73 35.73 0 0 1 1.4 5.11A7.48 7.48 0 0 0 19.42 13a12.76 12.76 0 0 0-4.57.06zM7.16 17.72A7.43 7.43 0 0 0 12 19.5a7.52 7.52 0 0 0 3.11-.68 35.06 35.06 0 0 0-1.49-5.47 12.78 12.78 0 0 0-6.46 4.37z"
                      fill="var(--color-gray5, #afb2be)"
                    />
                  </svg>
                </a>
              </li>
            </ul>
          </nav>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="sc-eqIVtm cQqwyE header"
      id="page-header"
    >
      <div
        class="layout__row header__row"
      >
        <a
          aria-current="page"
          aria-label="Press Enter/Click to go to the homepage."
          class="sc-cSHVUG ezoGnA"
          href="/"
          id="branding"
        >
          <svg
            class="sc-kAzzGY gQbpWy"
            height="48"
            id="logo"
            viewBox="0 0 48 48"
            width="48"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="brand__circle"
              cx="24"
              cy="24"
              fill="transparent"
              r="24"
            />
            <path
              class="brand__hair"
              d="M26.09 23.19s1.52-.6-.56 3.06c0 0 3.79-.87 3.11-3.94.22 1 .52.4-.12 3a4.73 4.73 0 0 0 2.65-2.65c0 .32.8.81-.33 2.9.56-.2.72-.39.85-.85a8.21 8.21 0 0 1-.43 3.74 5.18 5.18 0 0 0 1.67-2.3c.13-.92.77 3.37-.57 6.18a11.49 11.49 0 0 0 1.41-1.78c.56-1.22 0 2.84-.1 4.28-.49 2 .25 2 .25 2.92a3.73 3.73 0 0 0 1.59.12 2.87 2.87 0 0 0 1.83-1.34 6.75 6.75 0 0 0 .85-1.59.93.93 0 0 1-.61-.12c-.37-.24-.37-1-.37-1 2.07 1.15 2.37-.32 2.06-1.59a2 2 0 0 1-.27-1.44c.64-16.33-6.7-16.46-6.7-16.46a9 9 0 0 1 2.92-1.12 9.81 9.81 0 0 0-3.91 1.25 3.46 3.46 0 0 0-3-3.17A2.5 2.5 0 0 1 29.88 13s-5.53-4.33-11.46-1.84a9.4 9.4 0 0 1 4.77 0c-.19-.1-10.42-.95-14.55 8.13 1.6-1.77 2.15-2.4 2.46-2.16.19 0-2.23 3.58-2 12.93.15 1.47.5 1.25.73 1.71-.12.34-.35.4-.49.85-1.59 3 2.69 3.65 2.69 3.65a1.86 1.86 0 0 1-.37-2.44H12a2.15 2.15 0 0 1 .85.61c0-.37-.6-3.7 0-4.1 12.24-3.59 13.24-7.15 13.24-7.15zm-13.23 13.5a.6.6 0 0 1 .61-.12.62.62 0 0 1 .39.49v.12a3.81 3.81 0 0 1-2 .73c-.73 0-.59-.49-1.07-1.1.02 0 1.58.56 2.07-.12z"
            />
            <path
              class="brand__glasses"
              d="M12.11 27.91a.87.87 0 0 0 .26.7l.1.12a1.12 1.12 0 0 1 .12.35c.5 3 .61 3.42 1.87 4.08a7.57 7.57 0 0 0 3.09.53 7.89 7.89 0 0 0 3.52-.69c1.69-.88 2.33-3.94 2.36-4.09s.05-.38.83-.38c.73 0 .79.36.79.4s.66 3.18 2.35 4.07a8 8 0 0 0 3.6.7 7.56 7.56 0 0 0 3.08-.53c1.26-.66 1.37-1.07 1.87-4.08a1.12 1.12 0 0 1 .12-.35l.1-.12a.87.87 0 0 0 .26-.7V27a1 1 0 0 0-.85-.9 22.37 22.37 0 0 0-6-.3h-.11a8.09 8.09 0 0 0-3 .54l-.6.29a5.77 5.77 0 0 1-1.07.41l-.3.06a1.57 1.57 0 0 1-.53 0l-.3-.06a5.77 5.77 0 0 1-1.07-.41l-.6-.29a8.09 8.09 0 0 0-3-.54 22.37 22.37 0 0 0-6 .3 1 1 0 0 0-.85.9v.88zm23.79-.75v.45l-.56-.22zm-9.72.65c.08-.21.62-.91 3.93-1.1.58 0 1.07-.05 1.51-.05 2 0 2.58.35 2.95.92a3.26 3.26 0 0 1 .37 1.5c0 1-.19 2.73-1.55 3.38a6.12 6.12 0 0 1-2.5.48c-1.81 0-3.25-.61-3.87-1.64-1.29-2.16-.85-3.45-.84-3.48zm-12.24-.24c.37-.57 1-.92 3-.92.43 0 .92 0 1.51.05 3.31.19 3.85.89 3.94 1.13 0 0 .44 1.3-.85 3.46a4.44 4.44 0 0 1-3.87 1.64 6.12 6.12 0 0 1-2.5-.48c-1.36-.66-1.56-2.4-1.55-3.38a3.39 3.39 0 0 1 .32-1.5zm-1.34-.4l.56.23-.56.22z"
            />
          </svg>
        </a>
        <nav
          aria-label="Back Navigation Button"
          class="sc-bwzfXH iZKGMu"
          id="back-navigation"
        >
          <a
            aria-current="page"
            aria-label="Go back to the home page"
            class=""
            href="/"
          >
            <svg
              aria-labelledby="back-navigation-title"
              viewBox="0 0 48 48"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title
                id="back-navigation-title"
              >
                Navigate Back to the Homepage
              </title>
              <path
                class="back__navigation__wrapper"
                d="M0 0h48v48H0z"
                fill="none"
              />
              <path
                class="back__navigation__arrow"
                d="M40 22H15.66l11.17-11.17L24 8 8 24l16 16 2.83-2.83L15.66 26H40z"
                fill="var(--back-navigation-fill)"
              />
            </svg>
          </a>
        </nav>
        <h2
          class="sc-bdVaJa yzgbS"
          id="header-page-title"
        >
          Project Details
        </h2>
        <nav
          class="sc-fBuWsC enoLqX"
          id="top-navigation"
          title="Top Navigation"
        >
          <ul
            class="sc-fMiknA cHkchB"
          >
            <li
              class="sc-dVhcbM hWIHTP"
            >
              <a
                aria-label="Go to the about page"
                class="large-nav__link"
                href="/about/"
              >
                About
              </a>
            </li>
            <li
              class="sc-dVhcbM hWIHTP"
            >
              <a
                aria-label="Go to the contacts page"
                class="large-nav__link"
                href="/contact/"
              >
                Contact
              </a>
            </li>
          </ul>
        </nav>
        <nav
          class="sc-jlyJG iLFoRH"
          id="social-navigation"
          title="Social network links"
        >
          <ul
            class="sc-gipzik dClvSF"
          >
            <li
              class="sc-csuQGl exHRHf"
            >
              <a
                aria-label="Follow me on Facebook"
                class="sc-Rmtcm ghqkuW"
                href="https://fb.com/JoaoTMDias"
                rel="noopener noreferrer"
                tabindex="0"
                target="_blank"
                to="https://fb.com/JoaoTMDias"
              >
                <svg
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M12.88 22v-.07a10.55 10.55 0 0 1-1.13.07z"
                    fill="white"
                  />
                  <path
                    class="social__fill"
                    d="M15.12 13l1.13-2.81h-3.37V7.94A1.13 1.13 0 0 1 14 6.81h1.12V4h-2.24A3.38 3.38 0 0 0 9.5 7.38v2.81H7.25V13H9.5v8.74a10.41 10.41 0 0 0 2.25.26 10.55 10.55 0 0 0 1.13-.07V13z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                </svg>
              </a>
            </li>
            <li
              class="sc-csuQGl exHRHf"
            >
              <a
                aria-label="Follow me on Instagram"
                class="sc-Rmtcm ghqkuW"
                href="https://instagram.com/JoaoTMDias"
                rel="noopener noreferrer"
                tabindex="0"
                target="_blank"
                to="https://instagram.com/JoaoTMDias"
              >
                <svg
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="social__fill"
                    d="M15.75 2h-7.5A6.25 6.25 0 0 0 2 8.25v7.5A6.25 6.25 0 0 0 8.25 22h7.5A6.25 6.25 0 0 0 22 15.75v-7.5A6.25 6.25 0 0 0 15.75 2zm4.37 13.75a4.37 4.37 0 0 1-4.37 4.37h-7.5a4.37 4.37 0 0 1-4.37-4.37v-7.5a4.37 4.37 0 0 1 4.37-4.37h7.5a4.37 4.37 0 0 1 4.37 4.37z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                  <path
                    class="social__fill"
                    d="M12 7a5 5 0 1 0 5 5 5 5 0 0 0-5-5zm0 8.12A3.12 3.12 0 1 1 15.12 12 3.12 3.12 0 0 1 12 15.12z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                  <circle
                    cx="17.38"
                    cy="6.62"
                    fill="var(--color-gray5, #afb2be)"
                    r=".67"
                  />
                </svg>
              </a>
            </li>
            <li
              class="sc-csuQGl exHRHf"
            >
              <a
                aria-label="Check out my repos on Github"
                class="sc-Rmtcm ghqkuW"
                href="https://github.com/Joaotmdias"
                rel="noopener noreferrer"
                tabindex="0"
                target="_blank"
                to="https://github.com/Joaotmdias"
              >
                <svg
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="social__fill"
                    d="M12 2.21a10 10 0 0 0-3.16 19.56c.5.1.68-.21.68-.48v-1.71c-2.78.61-3.37-1.34-3.37-1.34A2.64 2.64 0 0 0 5 16.77c-.91-.62.07-.61.07-.61a2.1 2.1 0 0 1 1.53 1 2.13 2.13 0 0 0 2.94.84 2.22 2.22 0 0 1 .63-1.35c-2.22-.25-4.55-1.11-4.55-5A3.89 3.89 0 0 1 6.65 9a3.62 3.62 0 0 1 .1-2.66s.84-.27 2.75 1a9.92 9.92 0 0 1 2.5-.28 10 10 0 0 1 2.51.34c1.9-1.3 2.74-1 2.74-1a3.56 3.56 0 0 1 .1 2.6 3.89 3.89 0 0 1 1 2.69c0 3.86-2.34 4.71-4.57 5a2.39 2.39 0 0 1 .68 1.86v2.75c0 .33.18.58.69.48A10 10 0 0 0 12 2.21z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                </svg>
              </a>
            </li>
            <li
              class="sc-csuQGl exHRHf"
            >
              <a
                aria-label="Check out some dribble shots"
                class="sc-Rmtcm ghqkuW"
                href="https://dribbble.com/Joaotmdias"
                rel="noopener noreferrer"
                tabindex="0"
                target="_blank"
                to="https://dribbble.com/Joaotmdias"
              >
                <svg
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="social__fill"
                    d="M12 2a10 10 0 1 0 10 10A10 10 0 0 0 12 2zm0 18.75A8.75 8.75 0 1 1 20.75 12 8.77 8.77 0 0 1 12 20.75z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                  <path
                    class="social__fill"
                    d="M16.79 6.23A7.47 7.47 0 0 0 12 4.5a7 7 0 0 0-1.53.16 36 36 0 0 1 2.46 4 18 18 0 0 0 3.86-2.43zM14.39 11.88a13.87 13.87 0 0 1 5.1-.09 7.44 7.44 0 0 0-1.81-4.68 19 19 0 0 1-4.21 2.68c.33.69.64 1.38.92 2.09zM9.21 5a7.48 7.48 0 0 0-4.51 5.31 17.6 17.6 0 0 0 7.06-1.16A34.25 34.25 0 0 0 9.21 5zM12.33 10.29a18.92 18.92 0 0 1-6.83 1.3h-1v.45a7.46 7.46 0 0 0 1.77 4.83 14.05 14.05 0 0 1 6.9-4.64c-.26-.68-.54-1.32-.84-1.94zM14.85 13.06a35.73 35.73 0 0 1 1.4 5.11A7.48 7.48 0 0 0 19.42 13a12.76 12.76 0 0 0-4.57.06zM7.16 17.72A7.43 7.43 0 0 0 12 19.5a7.52 7.52 0 0 0 3.11-.68 35.06 35.06 0 0 0-1.49-5.47 12.78 12.78 0 0 0-6.46 4.37z"
                    fill="var(--color-gray5, #afb2be)"
                  />
                </svg>
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
